modules/RTC/TraceablePeerConnection.js

```diff
@@ -2134,14 +2135,24 @@ TraceablePeerConnection.prototype.setMaxBitRate = function() {
     if (this.isSimulcastOn()) {
         for (const encoding in parameters.encodings) {
             if (parameters.encodings.hasOwnProperty(encoding)) {
-                // On chromium, set a max bitrate of 500 Kbps for screenshare when
-                // capScreenshareBitrate is enabled through config.js and presenter
-                // is not turned on.
-                const bitrate = browser.usesPlanB()
-                    && videoType === VideoType.DESKTOP
-                    && this.options.capScreenshareBitrate
-                    ? presenterEnabled ? this.videoBitrates.high : DESKSTOP_SHARE_RATE
-                    : this.tpcUtils.localStreamEncodingsConfig[encoding].maxBitrate;
+                let bitrate;
+
+                if (planBScreenSharing) {
+                    // On chromium, set a max bitrate of 500 Kbps for screenshare when capScreenshareBitrate
+                    // is enabled through config.js and presenter is not turned on.
+                    // FIXME the top 'isSimulcastOn' condition is confusing for screensharing, because
+                    // if capScreenshareBitrate option is enabled then the simulcast is turned off
+                    bitrate = this.options.capScreenshareBitrate
+                        ? presenterEnabled ? this.videoBitrates.high : DESKSTOP_SHARE_RATE
+
+                        // Remove the bitrate config if not capScreenshareBitrate:
+                        // When switching from camera to desktop and videoQuality.maxBitratesVideo were set,
+                        // then the 'maxBitrate' setting must be cleared, because if simulcast is enabled for screen
+                        // and maxBitrates are set then Chrome will not send the screen stream (plan B).
+                        : undefined;
```

Comment:
The reason you think it breaks screensharing is because with the old logic, using capScreenshareBitrate=0 and specifying videoBitrates what happens is 
on encoding 0 - max. bitrate corresponding to LD (200 Kbps) gets applied.
on encoding 1 - max. bitrate corresponding to SD (700 Kbps) gets applied.
Since our SDP munging based simulcast produces only 2 encodings for screenshare, 0 with low fps and bitrate and 1 with high fps and high bitrate, the above max bitrates would not be enough to get the key frame for a desktop track out (esp if you are sharing the full screen) which makes it look like its broken.
Instead, can you apply SD and HD bitrates for encodings 0 and 1 and see if its fixes the screenshare ?

Reply:
It's not what I'm seeing. There are always 3 SSRCs produced regardless camera vs screenshare, but maybe I need to double check that. Now Chrome doesn't send screen if there are any bitrates set and there are multiple SSRCs for simulcast present. With `capScreenshareBitrate=0` and `maxBitrates` in config.js what happens is there are 3 maxBitrates values set which are the leftover from the camera. This is what I remember, but I will double check and try using the values you suggested. Thank you!

Reply:
Yes, there are 3 SSRC's because we munge SDP and add a total of 3 SSRCs for simulcast (same for camera and desktop tracks), but the bridge was receiving only 2 streams if I recall correctly. We don't know which encodings are the active ones looking at client stats since none of them are showing any valid values right now in chrome://webrtc-internals for simulcast case so its hard to tell. Yes, the same bitrates are applied to both camera and desktop tracks if `maxBitratesVideo` are set in config.js.

Reply:
This seems a bit similar to https://bugs.chromium.org/p/chromium/issues/detail?id=947495 -- chrome is behaving a bit weird at times

Reply:
@fippo there are so many broken cases like this, it will be great if we can get some clarity from the chrome team about how simulcast/non-simulcast screenshare is supposed to behave.

Reply:
It doesn't matter what bitrates I set it just doesn't send anything, as long as any values are set. The sender reports 0 resolution and nothing sent:

![Screen Shot 2020-10-08 at 7 09 47 PM](https://user-images.githubusercontent.com/2965063/95526239-152fcd80-099a-11eb-8cbc-01b605527e8d.png)

Tried this:

![Screen Shot 2020-10-08 at 7 08 54 PM](https://user-images.githubusercontent.com/2965063/95526348-1cef7200-099a-11eb-875e-0d69fe21391b.png)

and this:

![Screen Shot 2020-10-08 at 7 10 38 PM](https://user-images.githubusercontent.com/2965063/95526438-237de980-099a-11eb-8ff1-4ee83beb5b65.png)

no idea.

Clearing `maxBitrate` to `undefined` on every encoding works.

Reply:
Oh I saw the same thing with the webrtc-internals stats but the media was being sent and it was getting received on the other end. In any case let’s make it undefined for now as a workaround for the chrome bug.